tests/pdgid/test_pdgid.py

```diff
@@ -20,6 +20,11 @@ def test_class_operations(PDGIDs):
     assert id_positron == - id_electron
     assert PDGIDs.Positron == - id_electron
 
+def test_pdgint():
```

Comment:
What does "pdgint" hint at? You could have added this to the tests on operations that I defined in the previous PR. Not a problem, though, of course.

Reply:
This was written before the previous PR. This should be renamed, probably, but having more sections is a good thing.

Reply:
OK, I let you rename and then we're good to go.